yolox/data/datasets/coco.py

```diff
@@ -157,6 +158,9 @@ def load_image(self, index):
         image_file_path = self.image_paths[image_file_name]
 
         image = cv2.imread(image_file_path)
+        if image is None:
+            time.sleep(3)
+            image = cv2.imread(image_file_path)
         assert image is not None, f"file named {image_file_path} not found"
```

Comment:
I had often problems with loading images - probably issues with VPN/network drive connection. With this "fix" I could run full training without error.